app/javascript/packages/document-capture-polling/index.ts

```diff
@@ -70,12 +75,13 @@ export class DocumentCapturePolling {
    * @param {boolean} shouldPrompt Whether to bind or unbind page unload behavior.
    */
   bindPromptOnNavigate(shouldPrompt) {
-    window.onbeforeunload = shouldPrompt
-      ? (event) => {
-          event.preventDefault();
-          event.returnValue = '';
-        }
-      : null;
+    if (shouldPrompt && !this.promptOnNavigateBound) {
```

Comment:
is there a way to have `cleanUpPromptOnNavigate` be null/falsy/undefined, and use a null-safe navigator for calling ex `fn?.()` so we don't have to track a separate `promptOnNavigateBound` variable?